Google.Api.Generator/CodeGenerator.cs

```diff
@@ -236,6 +237,103 @@ private static void ValidateTransports(ApiTransports transports, List<ServiceDet
             }
         }
 
+        /// <summary>
+        /// Validate that all proto elements have comments (except the file descriptor).
+        /// We only log warnings for most missing elements; for services and methods, we log warnings
+        /// and then fail with an exception at the end, as the generated code won't compile.
```

Comment:
This is not the case as the code is at the moment. Maybe say something like 
```suggestion
        /// and in the future we may fail with an exception at the end, as the generated code won't compile.
```